client/src/components/Page.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { t } from "@lingui/macro";
+import { withI18n, withI18nProps } from "@lingui/react";
+import helpers from "../util/helpers";
+
+const metadata = {
+  keywords: t`Landlord, Portfolio, Tenant, Displacement, Map, JustFix, NYC, New York, Housing, Who Owns What`,
+  description: t`Use this free tool from JustFix.nyc to research your building and investigate landlords! We use property ownership mapping to identify a landlord's portfolio and provide data to indicate potential tenant harassment and displacement.`,
+  siteName: t`Who owns what in nyc?`,
+};
```

Comment:
Hey @toolness! Check out this implementation of the i18n of our page metadata— it utilizes lingui's notion of [Lazy Translations](https://lingui.js.org/tutorials/react-patterns.html?highlight=strings#lazy-translations). Was wondering if you ever saw this and have concerns about this pattern, as the conciseness of it almost seems too good to be true. That being said, everything seems to be behaving as it should in terms of these strings getting translated 🤷‍♂️

Reply:
oh this is super dope and actually makes the whole `i18n._(t`blah`)` thing make a lot more sense (I always thought it was kind of verbose).  NOICE